src/test/java/it/com/atlassian/bitbucket/jenkins/internal/status/BuildStatusPosterIT.java

```diff
@@ -253,24 +251,27 @@ private static RequestPatternBuilder requestBody(RequestPatternBuilder requestPa
         assertNotNull(bitbucketRevisionAction);
         String refName = bitbucketRevisionAction.getBranchAsRefFormat();
         String jenkinsUrlAsString = jenkinsUrl.toExternalForm();
+        ItemGroup<?> parentProject = job.getParent();
+        String parentString = parentProject instanceof MultiBranchProject ? parentProject.getFullName() : job.getFullName();
```

Comment:
nit: `parentString` is a bit vague and redundant (we can see the type is `String` ;-)
`parentName`? Just `parent`? Anything but `parentString` please!

Reply:
Haha I was copying the form of `jenkinsUrlAsString` but it felt weird when I typed it. Lets go with `parentName` :)